### PR TITLE
fix: error message for entering less than 4 characters for username

### LINKED ISSUE #603: Error message for entering less than 4 characters for username  under update user profile.
When we enter username less than 4 character, the error message should be listed under Responses.

**To Reproduce**

1. Go to 'user-update user profile'
2. provide authorization access token
3. {
  "name": "123”,
  "username": "123",
  "bio": "string",
  "occupation": "string",
  "organization": "string",
  "slack_username": "string",
  "social_media_links": "string",
  "skills": "string",
  "interests": "string",
  "resume_url": "string",
  "photo_url": "string",
  "need_mentoring": false,
  "available_to_mentor": false,
  "location": "string"
}

4.  Click on 'Execute'
5. Scroll down to 'Responses'
6. 
Screenshot (81)

**Expected behavior**
That error message should be listed in the responses.

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser  chrome

### PR DESCRIPTION
**Pull request by priyanshuone6**
### Description
Fixed Error message for entering less than 4 characters for username under the update user profile and listed the error in the code section

Fixes #603 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Screenshot (32)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

File: app/api/resources/user.py [hunk 1]
```diff
@@ -129,7 +129,14 @@ def get(cls):
...
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_SUCCESSFULLY_UPDATED)
-    @users_ns.response(HTTPStatus.BAD_REQUEST, "Invalid input.")
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST,
+    "%s\n%s"
+    % (
+         "Invalid input.",
+         messages.USERNAME_FIELD_IS_EMPTY
+    ),
+)
```
Comment (SanketDG) [inline app/api/resources/user.py]: Prettifying this makes this more unreadable :)

let's just go with the orignal style.

**Review by SanketDG — CHANGES_REQUESTED** [review]

**Comment by priyanshuone6** [comment]
@SanketDG I have changed the style, please review it and let me know

**Review by vj-codes — CHANGES_REQUESTED** [review]
@priyanshuone6 can you please update the `how has this been tested part` .
I see the changes in HTTP.status Bad_request , but the response on top still shows longer than 4 and shorter than 26.
Maybe upload 2 screenshots one containing dummy data and second of responses.What say?

**Comment by priyanshuone6** [comment]
> @priyanshuone6 can you please update the `how has this been tested part` .
> I see the changes in HTTP.status Bad_request , but the response on top still shows longer than 4 and shorter than 26.
> Maybe upload 2 screenshots one containing dummy data and second of responses.What say?

@vj-codes I just added `messages.USERNAME_FIELD_IS_EMPTY` in `users_ns.response` and changed nothing except it.
Please have a look at the screenshots

Screenshot (41)
Screenshot (42)
Screenshot (43)

**Review by vj-codes — CHANGES_REQUESTED** [review]
@priyanshuone6 hey please solve the merge conflicts.Thanks!

**Review by Aaishpra — CHANGES_REQUESTED** [review]
@priyanshuone6 can you remove the merge conflicts(as suggested above) and run black app/api/resources/user.py to reformat the code according to black standards so that the failing test can pass.

**Comment by epicadk** [comment]
The tests are failing can you look into that @priyanshuone6

**Comment by vj-codes** [comment]
@priyanshuone6 you've closed this PR , let us know if you would like to continue on this issue orthwise it'll be marked as available for other contributors.

**Comment by priyanshuone6** [comment]
This PR is open for about 5 months, due to which there are merge conflicts arising and I have solved them twice and no one's responding so I closed the PR.
You can mark the issue as open.

**Pull request closed without merging**